src/test/java/org/apache/rocketmq/dashboard/controller/ConsumerControllerTest.java

```diff
@@ -182,6 +182,8 @@ public void testDelete() throws Exception {
         final String url = "/consumer/deleteSubGroup.do";
         {
             doNothing().when(mqAdminExt).deleteSubscriptionGroup(any(), anyString());
+            doNothing().when(mqAdminExt).deleteTopicInBroker(any(), anyString());
```

Comment:
Do we have a better operation than do-nothing when delete happend?

Reply:
In unit tests, all RPC interfaces involved in interacting with RocketMQ used mocks for data return，and this two delete interface methods return void

Reply:
So, how do we know we have deleted what we wanted topic, do we need to check here?

Reply:
No need here，mock these two delete interfaces indicate that the topic has been deleted successfully on the broker side, the unit test only need to care about the code logic in the dashboard before and after the deletion.